stdlib/LinearAlgebra/test/svd.jl

```diff
@@ -92,6 +92,18 @@ a2img  = randn(n,n)/2
             b = rand(eltya,n)
             @test usv\b ≈ transpose(a)\b
         end
+        @testset "singular value decomposition of Hermitian/real-Symmetric" begin
```

Comment:
Also seems like this should be outside the `for (a, a2)` loop since it only uses `asym`

Reply:
Good catch! Actually, that loop is mysterious. `a2` is never used in it. I should still move the test outside, but should I also simplify that loop? Or leave it until someone guesses what `a2` was meant to be good for and includes it in tests?

Reply:
Simplifying that loop would be good.

Reply:
There are a couple of other unused variables. I wonder about the following: after setting `Random.seed!(1234321)`, and then calling
```julia
areal = randn(n,n)/2
aimg  = randn(n,n)/2
a2real = randn(n,n)/2 # unused
a2img  = randn(n,n)/2 # unused
```
the latter two are not used, anywhere. If I remove them, does that change the random numbers created in subsequent `rand*` calls? Probably yes. But is that an issue? Should I leave them for test consistency? I don't think that we rely here on that the random numbers in future tests are exactly the same as they used to be, but I'm not sure how strictly that is handled.

Reply:
> Should I leave them for test consistency?

No, it's ok to change them, but if tests start failing it may require using a different seed or loosening some approximate equality tests. The main reason for using a fixed seed is so that we don't get random failures during CI, but if the bounds are good then most seeds should be fine.

Reply:
Tests pass locally.